plio/io/tests/test_io_controlnetwork.py

```diff
@@ -31,6 +31,38 @@ def test_cnet_read(cnet_file):
         assert proto_field not in df.columns
         assert mangled_field in df.columns
 
+@pytest.mark.parametrize('messagetype, value', [
+                         (2, 0.5),
+                         (3, 0.5),
+                         (4, -0.25),
+                         (5, 1e6),
+                         (6, 1),
+                         (7, -1e10),
+                         ('GoodnessOfFit', 0.5),
+                         ('MinimumPixelZScore', 0.25)
+])
+def test_Log(messagetype, value):
+    l = io_controlnetwork.Log(messagetype, value)
+    if isinstance(messagetype, int):
+        assert l.messagetype == io_controlnetwork.MessageType(messagetype)
+    elif isinstance(messagetype, str):
+        assert l.messagetype == io_controlnetwork.MessageType[messagetype]
+        
+    assert l.value == value
+    assert isinstance(l.to_protobuf, object)
+
+def test_log_error():
+    with pytest.raises(TypeError) as err:
+        io_controlnetwork.Log(2, 'foo')
+
+def test_to_protobuf():
+    value = 1.25
+    int_dtype = 2
+    log = io_controlnetwork.Log(int_dtype, value)
+    proto = log.to_protobuf()
+    assert proto.doubleDataType == int_dtype
+    assert proto.doubleDataValue == value
+
```

Comment:
We also need to test reading and writing a control network that contains measure log data.

Reply:
@jessemapel :+1:

@ladoramkershner I'm updating to use pytest while doing this since the test now isn't working how we think it is. So I won't address this until this afternoon.

Reply:
I don't see a test for reading in a network. Is that because we would need an actual control network as test data?

Reply:
All the tests read in the network that was written out in the fixture. So these are all read tests. The 'write test' is the fixture, which will throw an error (and not a failure) if it has a problem. I am torn between shipping a canned network and testing round trip this way. 

Arguably we should have ISIS read one of these networks, but that is not going to happen right now because ISIS, I believe, wants to load image data and sensor models which require way too much data. Hopefully someone can weigh in if I am wrong!